Pidaq/UI/DTS/dts_dash/layout/control.py

```diff
@@ -0,0 +1,305 @@
+                        type="number",
+                        min=0,
+                        max=10000, 
+                        step=100,
+                        value=0,
+                        style={"text-align": "center", "width": "80%", "margin": "auto"},
+                        className="vertical-center")
+                ])
+            ],
+            className="custom-card")
+        ),
+        
+        dbc.Col([
+            dbc.Card([
+                dbc.CardHeader("RPM / Torque Value", style={"text-align": "center"}),
+                dbc.CardBody([
+                    dbc.InputGroup([
+                        dbc.Input(type="number",
+                                  id="rpm-torque-value",
+                                  step=100,
+                                  min=0,
+                                  max=10000,
+                                  value=0),
+                        dbc.InputGroupAddon(add_command_btn,
+                                            addon_type="append")
+                    ])
+                ])
+            ],
+            className="custom-card"),  
+        ],
+        className="pad-right"),
+    ]),
+    
+    
+    dbc.Row([
+        dbc.Col([
+            dbc.Card([
+                dbc.CardHeader([
+                    "Time-Step Readout of Test Torque/RPM Input Profile",
+                    export_profile_btn
+                    ], 
+                    style={"text-align": "center"}),
+                
+                dbc.CardBody([
+                    timestep_readout_table
+                ])
+            ], 
+            className="pad-right pad-left")
+        ])
+    ],
+    className="pad-top pad-bot"),
+    
+    
+    dbc.Row([
+        dbc.Col([
+            dbc.Card([
+                dbc.CardBody([
+                    html.Div([
+                       start_test_btn
+                    ]),
+                    html.Div([
+                       stop_test_btn
+                    ])
+                ])
+            ], className="custom-card")
+        ],
+        className="pad-left"),
+        
+        dbc.Col([
+            dbc.Card([
+                dbc.CardBody([
+                   estop_btn
+                ])
+            ], className="custom-card")
+        ],
+        width={"size": 5}),
+        
+        dbc.Col([
+            dbc.Card([
+                dbc.CardBody([
+                    html.Div([
+                       clear_last_btn
+                    ]),
+                    html.Div([
+                       clear_all_btn
+                    ])
+                ])
+            ], className="custom-card")
+        ],
+        className="pad-right")
+    ],
+    className="pad-top pad-bot"),
+    
+    # Empty divs to store output of callbacks with no output
+    html.Div("", id="dump-abort-start"),
+    html.Div("", id="dump-export-profile"),
+    html.Div("", id="dump-start-clicked")
+])
```

Comment:
- Gotta add a newline at EOF
- Some of the gross inline styles are in here because it wasnt working when I used classes to update the same css attributes, it was so weird. The most prominent example is the rpm/torque button group positioning.
- For line spacing in the bootstrap layout, I went with 2 spaces between rows and 1 between columns. I can go back to one line between each row though - just thought it might allow you to easier distinguish between rows and result in better readability